plugins/com.google.cloud.tools.eclipse.appengine.validation.test/META-INF/MANIFEST.MF

```diff
@@ -8,7 +8,8 @@ Bundle-Vendor: Google Inc.
 Bundle-Version: 0.1.0.qualifier
 Fragment-Host: com.google.cloud.tools.eclipse.appengine.validation
 Require-Bundle: org.junit,
- org.eclipse.equinox.registry
+ org.eclipse.equinox.registry,
+ com.google.cloud.tools.eclipse.test.dependencies
```

Comment:
Import-Package works when you're using the API.  Require-Bundle is when you depend on the implementation.  When we're using `provider=google', we're implicitly using the implementation.

On the flip side, `Import-Package` helps pare down dependencies since you have to explicitly bring them in.  It helps avoid problems from kitchen-sink type bundles (like our `.test.dependencies`, but also `org.eclipse.core.runtime`) that pull in the world.

So I'm +1 to using Import-Package here for the Mockito deps.